### DISCUSSION TITLE
Using scenic plus on multiome

### DISCUSSION THREAD

**Discussion by RosaDeSa** [aertslab/scenicplus#197] [Q&A]
I have an analysis of 10X multi-omic data (joint RNA and ATAC analysis) with (Signac/Seurat).  Can I still use SCENIC+ to perform a GRN? I thought to import the assay of GEX and the assay of ATAC separately to create a cistopic object. That could be correct?
 Thx

**Comment by SeppeDeWinter** [comment]
Hi @RosaDeSa 

Yes, you can import your data in Python.

See: https://pycistopic.readthedocs.io/en/latest/faqs.html

Best,

Seppe

**Reply by cesarsierran** [reply]
Hi @SeppeDeWinter ,

Thank you for the amazing tool. I am interested in this same question but the link provided inside the FAQs to explain how to create a cisTopic object from a count matrix seems to be broken.

**Reply by SeppeDeWinter** [reply]
Hi @cesarsierran 

Thank you!
You can still find this notebook here: https://github.com/aertslab/pycisTopic/blob/old/notebooks/Toy_melanoma-RTD.ipynb

Best,

Seppe
